tests/features/joinup_core/unpublished_entities.feature

```diff
@@ -142,3 +142,71 @@ Feature: Unpublished content of the website
+    And users:
+      | Username  | Roles |
+      | Ed Abbott |       |
+    And the following solutions:
+      | title               | description         | state     | owner           | contact information |
+      | Invisible Boyfriend | Invisible Boyfriend | validated | Owner something | Published contact   |
+    And the following solution user memberships:
+      | solution            | user      | roles         |
+      | Invisible Boyfriend | Ed Abbott | authenticated |
+    And "event" content:
+      | title                | created           | author    | solution            | state     |
+      | The Ragged Streams   | 2018-10-04 8:31am | Ed Abbott | Invisible Boyfriend | proposed  |
+      | Storms of Touch      | 2018-10-04 8:31am | Ed Abbott | Invisible Boyfriend | validated |
+      | The Male of the Gift | 2018-10-04 8:31am | Ed Abbott | Invisible Boyfriend | validated |
+      | Mists in the Thought | 2018-10-04 8:31am | Ed Abbott | Invisible Boyfriend | draft     |
+    And the following releases:
+      | title        | documentation | release number | release notes | creation date    | is version of       | state |
+      | Hidden spies | text.pdf      | 3              | Notes 3       | 28-01-1995 12:05 | Invisible Boyfriend | draft |
+
+    # The owner should be able to see all content.
+    When I am logged in as "Ed Abbott"
+    And I go to the "Invisible Boyfriend" solution
+    Then I should see the "The Ragged Streams" tile
+    And I should see the "Storms of Touch" tile
+    And I should see the "The Male of the Gift" tile
+    And I should see the "Mists in the Thought" tile
+    # The owner of the content did not create the release as they are simple members.
+    And I should not see the following tiles in the "Unpublished content area" region:
+      | Hidden spies |
+
+    # Other authenticated users should only see the published items.
+    When I am logged in as a user with the "authenticated" role
+    And I go to the "Invisible Boyfriend" solution
+    Then I should see the "Storms of Touch" tile
+    And I should see the "The Male of the Gift" tile
+    But I should not see the "The Ragged Streams" tile
+    And I should not see the "Mists in the Thought" tile
+    And I should not see the following tiles in the "Unpublished content area" region:
+      | Hidden spies |
+
+    # The facilitator should not be able to see content that only have a draft state.
+    When I am logged in as a facilitator of the "Invisible Boyfriend" solution
+    And I go to the "Invisible Boyfriend" solution
+    Then I should see the "The Ragged Streams" tile
+    And I should see the "Storms of Touch" tile
+    And I should see the "The Male of the Gift" tile
+    But I should not see the "Mists in the Thought" tile
+    # Releases are available.
+    And I should see the following tiles in the "Unpublished content area" region:
+      | The Ragged Streams |
+      | Hidden spies       |
```

Comment:
This doesn't seem to be correct, the ticket does not mention that facilitators should be able to see unpublished drafts of releases. It goes against the established pattern "The facilitator should not be able to see content that only have a draft state".

It is requested that authors should see their own releases, not facilitators.